src/lib/file_icons.ts

```diff
@@ -49,6 +50,7 @@ const file_icons = new Map<RegExp, typeof File>([
 	[/^postcss\.(js|cjs|mjs)$/, PostCSS],
 	[/.*\.postcss$/, PostCSS],
 	[/.*\.(js|cjs|mjs)$/, JS],
+	[/.*\.(jpe?g|png|gif|bmp)$/, Images],
```

Comment:
we'd wanna change this too if we add .svg and .webp

Reply:
Here's i'll add svg and webp too